SCons/SConf.py

```diff
@@ -253,8 +253,7 @@ def failed(self):
         # ConfigureCacheError and if yes, reraise the exception
         exc_type = self.exc_info()[0]
         if issubclass(exc_type, SConfError):
-            # TODO pylint E0704: bare raise not inside except
-            raise
+            raise self.exc_info()[1]
```

Comment:
can you add a comment what this is?

Reply:
I will if you think it is necessary but I sort of think the line up on 254 and the comment above it make it reasonably clear already. Actually clearer now because now both cases are acting on the state in `self.exc_info()`.

Reply:
@bdbaddog - What do you think? Does it need more commentary?

Reply:
Style note: normally, I'd grumble about calling a function/method twice right by each other, although in this case the `exc_info` method is a trivial one-liner returning an attribute. Seems like you could, nonetheless, unpack the exception tuple in the first call instead of having to use `[0]` and `[1]` just for readability.

Reply:
@acmorrow - nope. you're right 253's comment is sufficient